pkgs/applications/networking/cluster/helm/plugins/helm-secrets.nix

```diff
@@ -2,7 +2,7 @@
 
 stdenv.mkDerivation rec {
   pname = "helm-secrets";
-  version = "3.7.0";
+  version = "3.8.1";
 
   src = fetchFromGitHub {
     owner = "jkroepke";
```

Comment:
Missing hash update.

Reply:
For some reason my machine gave me the same value. I'll try again.

Reply:
I had to use nix-prefetch-github. Weird.

Reply:
common issue with FOD, hash defines it, not anything else. You need to invalidate it to get a new one